R/epidatacall.R

```diff
@@ -334,37 +268,3 @@ request_impl <- function(epidata_call, format_type, fields = NULL) {
 
   response
 }
-
-#' @export
-print.epidata_csv <- function(x, ...) {
-  char_limit <- getOption("epidata_csv__char_limit", default = 300L)
-  cat(
-    "# A epidata_csv object with", nchar(x), "characters; showing up to", char_limit,
-    "characters below. To print the entire string, use `print(as.character(x))`:\n"
-  )
-  cat(substr(x, 1L, char_limit))
-  if (nchar(x) > char_limit) {
-    cat("[...]")
-  }
-  cat("\n")
-  invisible(x)
-}
-
-info_to_type <- function(info, disable_date_parsing = FALSE) {
-  types <- list(
-    date = if (disable_date_parsing) {
-      readr::col_integer()
-    } else {
-      readr::col_date(format = "%Y%m%d")
-    },
-    epiweek = readr::col_integer(),
-    bool = readr::col_logical(),
-    text = readr::col_character(),
-    int = readr::col_integer(),
-    float = readr::col_double(),
-    categorical = readr::col_factor(info$categories)
-  )
-  r <- types[info$type]
-  stopifnot(!is.null(r))
-  r
-}
```

Comment:
praise: nice catch eliminating this unused code.  Pretty exciting to see some complexity go away!

Reply:
it's so cathartic deleting code :D